src/keypair.ts

```diff
@@ -204,14 +217,24 @@ export class Keypair {
    * @param {Buffer} signature Signature
    * @returns {boolean}
    */
-  verify(data, signature) {
+  public verify(data: Buffer, signature: Buffer): boolean {
     return verify(data, signature, this._publicKey);
   }
 
-  signDecorated(data) {
+  public signDecorated(data: Buffer) {
```

Comment:
@Akuukis for the return value, check https://github.com/stellar/js-stellar-base/blob/typescript/src/%40types/xdr.ts#L28 --- see https://github.com/stellar/js-stellar-base/blob/typescript/src/operations/index.ts#L5

You can do  `import { xdr as xdrDef } from '../@types/xdr';`